cns/restserver/internalapi_test.go

```diff
@@ -30,6 +30,10 @@ import (
 
 const (
 	primaryIp           = "10.0.0.5"
+	SWIFTv2IP           = "192.168.0.1"
+	SWIFTv2MAC          = "00:00:00:00:00:00"
+	SWIFTv2GatewayIP    = "10.0.0.1"
+	SWIFTv2NCID         = "testncid"
```

Comment:
Looks unused

Reply:
Perhaps this is why build is failing against your private fork? 
Also, do we have a tool to check these kind of issues locally?

Reply:
yeah it's weird how the linter did not catch this, `make lint` always run before I can push anything to the branch.

Reply:
and no, I don't think build should be failing because of unused constant.

Reply:
I see this is used in ipam_test.go, so it might not be the issue for build failing